app/client/src/pages/_app.tsx

```diff
@@ -11,13 +11,31 @@ import { UserProvider } from '@local/features/accounts/UserContext';
 import { ThemeProvider, SnackContext, useEnvironment } from '@local/features/core';
 import { Layout } from '@local/layout';
 import '@local/index.css';
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
```

Comment:
Please use `React.useEffect` not just useEffect, we already have it imported and this should be giving you a lint error.